server/routes/quiz.js

```diff
@@ -0,0 +1,85 @@
+const express = require('express');
+const router = express.Router();
+const quizLibs = require('../libs/quizlibs');
+const {authenticate} = require('./../middleware/authenticate');
+
+router.post('/quiz/createQuiz', (req, res) => {
+    quizLibs.createQuiz(req.body).then((quiz) => {
+        res.status(quiz.status).send(quiz);
+    }).catch((error) => {
+        res.status(error.status).send(error);
+    });
+});
+
+router.get('/quiz/getQuizById/:qid', (req, res) => {
+    let query = {
+        qid : req.params.qid
+    }
+    quizLibs.getQuiz(query).then((quiz) => {
+        res.status(quiz.status).send(quiz);
+    }).catch((error) => {
+        res.status(error.status).send(error);
+    });
+});
+
+router.post('/quiz/addQuestion', (req, res) => {
+    quizLibs.addQuestion(req.body).then((quiz) => {
+        res.status(quiz.status).send(quiz);
+    }).catch((error) => {
+        res.status(error.status).send(error);
+    });
+});
+
+router.get('/quiz/getQuestionByType', (req, res) => {
```

Comment:
should be in params, for get request using any wehere in the application its should be in params

Reply:
Done